src/DigitalProductPassport/Battery/ManufacturingDataSheet_v0.1.py

```diff
@@ -394,12 +394,12 @@ class ManufacturingDataSheetRequest(CamelCaseModel):
 
 
 DEFINITION = DataProductDefinition(
-    version="0.1.5",
+    version="0.1.4",
```

Comment:
We shouldn't downgrade this definition. Looks also like all the changes between the two versions are such that the 0.1.5 version is the better one. It looks like the main bulk of changes was actually done in the commit `Description changes and tags for definitions (#92)` (`3f1d78b90527aff42269812abe000fec0a52d67d`). Not sure why the same changes were not at the same time done to the ioxio-io-definitions.

Do you think you could revert the changes to this file and make a PR to the ioxio-io-definitions to do the same updates to this file in that repo?